ansible/roles/rhn-subscription-manager/tasks/main.yml

```diff
@@ -23,14 +23,19 @@
   tags:
     - subscription  
 
-
 - name: Disable all repos by default
-  command: subscription-manager repos --disable "*"
+#   command: subscription-manager repos --disable "*"
+  rhsm_repository:
+    name: "*"
+    state: disabled
   tags:
     - subscription
```

Comment:
Any reason why you're not using `set-repositories` role with `repo_method==rhn` instead of creating `rhn-subscription-manager` ?
https://github.com/redhat-cop/agnosticd/blob/development/ansible/roles/set-repositories/tasks/rhn-repos.yml

Reply:
Did not notice the capability of set-repositoies role but now changed